src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -249,12 +264,39 @@ private static string GetFileName(Instance instance, TextResource? textResource)
         return null;
     }
 
+    private static string? GetPdfPreviewText(TextResource? textResource)
+    {
+        if (textResource is not null)
+        {
+            TextResourceElement? titleText = textResource.Resources.Find(textResourceElement =>
+                textResourceElement.Id.Equals("pdfPreviewText", StringComparison.Ordinal)
+            );
+
+            if (titleText is not null)
+            {
+                return titleText.Value;
+            }
+        }
+
+        return null;
```

Comment:
Should the default `"Dette er en forhåndsvisning"` be set here instead of in the `GetPreviewFooter`?